scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -19,10 +19,10 @@ const customDirectories = (baseDirectory, directories) =>
 
 const isFile = filePath => (fse.existsSync(filePath) && !fse.lstatSync(filePath).isDirectory());
 
-const configFile = (configPath) => {
+const loadConfigFile = (configPath) => {
   if (configPath) {
     // eslint-disable-next-line global-require, import/no-dynamic-require
-    return require(configPath);
+    return require(path.resolve(process.cwd(), configPath));
```

Comment:
I don't think we should make this change. We should leave the require behavior as is. If someone wants a relative path they can use `./config`.